api/v1/README.md

```diff
@@ -458,6 +458,8 @@ https://github.com/opencontainers/runtime-spec/blob/main/config.md#createcontain
 | mount | [string](#string) |  |  |
 | path | [string](#string) |  |  |
 | flags | [string](#string) |  |  |
+| inode | [uint64](#uint64) |  |  |
```

Comment:
Thank you for this feature, so I'm thinking if the inode needs more context to be really useful in other kprobe use cases, not only this one. The dev as an integer doesn't make that much sense to users, something like /proc/self/mountinfo is probably good. We have some context about the file itself, maybe we need same about the inode and its backing store, so users can grock it.

How about:
```
"inode": {
      "number": "1",
      "fs": {
            "name": "ext4",
            "dev": "1:1"
       }
}
```

So from this it should be something like:
```
message FileSystem {
    string name = 1;   // name of the filesystem
    string dev = 2;
}

message Inode {
    uint64 number = 1;
    FileSystem fs = 2;
}
```

The filesystem allows us to augment with the super block and fscontext easily in future... we can put this into its own file.proto maybe? @tpapagian 

Let's see what the others think about it

Reply:
I believe that having an inode number and device ID is enough for everything. We can introduce other fields like file system etc. but this can also happen as part of a next PR.

What I would suggest is that having a device number as it is now makes things a bit more difficult to interpret for users. Generally, the device is in the form of `<major:minor>`. Changing `uint32` to `string` and printing something like `"1:1"` would be great. Another approach would be to have separate fields (i.e. `device_major` and `device_minor` both as `uint32`) but I would say that I prefer the `string` approach. 

We can also have separate messages for that like what Djalal suggested. i.e.:
```
message FileDetails {
  uint64 number = 1;
  string device = 2;
};
```
and reuse that in `KprobePath` and `KprobeFile`. And we can extend that later if we need to do so.